tests/prague/eip6206_jumpf/test_jumpf_target.py

```diff
@@ -0,0 +1,100 @@
+"""
+EOF JUMPF tests covering JUMPF target rules.
+"""
+
+import pytest
+
+from ethereum_test_tools import EOFException, EOFTestFiller, StateTestFiller
+from ethereum_test_tools.eof.v1 import Container, Section
+from ethereum_test_tools.eof.v1.constants import NON_RETURNING_SECTION
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+from .helpers import execute_tests
+from .spec import EOF_FORK_NAME
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-6206.md"
+REFERENCE_SPEC_VERSION = "a1775816657df4093787fb9fe83c2f7cc17ecf47"
+
+pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
+
+
+@pytest.mark.parametrize(
+    "target_outputs",
+    [NON_RETURNING_SECTION, 0, 2, 4],
+    ids=lambda x: "to-%s" % ("N" if x == NON_RETURNING_SECTION else x),
+)
+@pytest.mark.parametrize(
+    "current_outputs",
+    [NON_RETURNING_SECTION, 0, 2, 4],
+    ids=lambda x: "co-%s" % ("N" if x == NON_RETURNING_SECTION else x),
+)
+def test_jumpf_target_rules(
+    state_test: StateTestFiller,
+    eof_test: EOFTestFiller,
+    current_outputs: int,
+    target_outputs: int,
+):
+    """
+    Validate the target section rules of JUMPF, and execute valid cases.
+    We are not testing stack so a lot of the logic is to get correct stack values.
+    """
+    current_non_returning = current_outputs == NON_RETURNING_SECTION
+    current_height = 0 if current_non_returning else current_outputs
+
+    target_non_returning = target_outputs == NON_RETURNING_SECTION
+    target_height = 0 if target_non_returning else target_outputs
+
+    delta = 0 if target_non_returning or current_non_returning else target_outputs - current_height
```

Comment:
can you add more comments what is the meaning of delta and current/target in text also?

Reply:
Done.